src/insights/competition.ts

```diff
@@ -62,6 +62,21 @@ export interface PNLLeaderboardEntry {
   twitter: string | null;
 }
 
+// Lottery Rankings
+export interface LotteryLeaderboardEntry {
+  rank: string;
+  competitionId: string;
+  address: string;
+  twitter: string;
+  username: string;
+  longestPosition: string;
+  maxLeverage: string;
+  referralTickets: string;
+  liquidationTickets: string;
+  positionTickets: string;
+  totalTickets: string;
```

Comment:
Don't forget to run the [API query](https://test-api-insights.carbon.network/competition/leaderboardlottery?competitionId=tradingcomp7) to double-check your struct, in this case, `twitter` and `username` can be `null`
<img width="606" alt="Screenshot 2023-04-04 at 1 46 18 PM" src="https://user-images.githubusercontent.com/65583522/229699780-12350230-8ad6-490c-9e80-3bb26bb443a2.png">